packages/@ember/-internals/views/lib/system/utils.js

```diff
@@ -60,23 +60,30 @@ export function getViewId(view) {
   }
 }
 
-const VIEW_ELEMENT = symbol('VIEW_ELEMENT');
+const VIEW_ELEMENT = new WeakMap();
+const ELEMENT_VIEW = new WeakMap();
 
 /**
   @private
   @method getViewElement
   @param {Ember.View} view
  */
 export function getViewElement(view) {
-  return view[VIEW_ELEMENT];
+  return VIEW_ELEMENT.get(view) || null;
 }
 
-export function initViewElement(view) {
-  view[VIEW_ELEMENT] = null;
+export function getElementView(element) {
+  return ELEMENT_VIEW.get(element) || null;
 }
 
 export function setViewElement(view, element) {
-  return (view[VIEW_ELEMENT] = element);
+  VIEW_ELEMENT.set(view, element);
+  ELEMENT_VIEW.set(element, view);
+}
+
+// Not needed for GC, but for correctness
+export function clearViewElement(view) {
```

Comment:
Shouldn't this also remove from `ELEMENT_VIEW`? Seems _slightly_ odd to have this mismatch vs setViewElement